test/support/foreman_test_helper_additions.rb

```diff
@@ -0,0 +1,21 @@
+require "#{ForemanTemplates::Engine.root}/test/support/fixture_support"
```

Comment:
These two files can probably go away if you use factories too - since the factory is in core, you can just use it directly in tests, eg https://github.com/theforeman/foreman_default_hostgroup/blob/develop/test/unit/default_hostgroup_test.rb#L15-L17